test/openshift/e2e/sequential/1-056_validate_managed-by/05-check.yaml

```diff
@@ -2,7 +2,15 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
 - script: |
-    if [ -z $CI ]; then
+    if ! [ -z $NON_OLM ]; then
+      label_value="true"
+      if [ -z $label_value ]; then
+        echo "REMOVE_MANAGED_BY_LABEL_ON_ARGOCD_DELETION not set"
+        exit 1
+      else
+        exit 0
+      fi
```

Comment:
Same here.. This `$label_value` check does not make sense. We should retrieve the value first.